client/lib/analytics/index.js

```diff
@@ -37,7 +38,61 @@ window.ga = window.ga || function() {
 };
 window.ga.l = +new Date();
 
-loadScript( '//stats.wp.com/w.js?56' ); // W_JS_VER
+function getUrlParameter( name ) {
+	name = name.replace( /[\[]/, '\\[' ).replace( /[\]]/, '\\]' );
+	const regex = new RegExp( '[\\?&]' + name + '=([^&#]*)' );
+	const results = regex.exec( location.search );
+	return results === null ? '' : decodeURIComponent( results[ 1 ].replace( /\+/g, ' ' ) );
```

Comment:
particularly when using members of an array as results I like to bring in destructuring where possible. in this case, I think we could imagine a rearrangement of this line into a few and increase readability

```js
const querySearch = namePattern.exec( location.search );

if ( ! querySearch ) {
	return '';
}

const [ /* full match */, queryValue ] = querySearch;
return decodeURIComponent( querySearch.replace( … ) );
```